Kajabi/sage-lib
# Fixes 597 - Update `help_link` attribute to use `name`

Closes #597 

## Description

In [code examples in the design system](https://sage-design-system.kajabi.com/pages/component/page_heading) as well as throughout `kajabi-products`, the `name` attribute is being used within the `help_link` object. Per the structure of the component, `name` is incorrect. `label` is the correct key, therefore prior to this PR, the `name` value does not output into the visually hidden span.

**Design System Code Example**
![Screen Shot 2021-06-28 at 10 19 02 AM](https://user-images.githubusercontent.com/24237393/123661822-7afb0880-d7fa-11eb-93e8-5dd4df251fb4.png)

**Component code**
![Screen Shot 2021-06-28 at 10 21 04 AM](https://user-images.githubusercontent.com/24237393/123661972-9ebe4e80-d7fa-11eb-9428-8d00587cb1fa.png)


Since `kajabi-products` is using `name` widely, update `sage-lib` to using `name` instead of `label` for the `help_link` accessible description.


## Screenshots
| Before Update | After Update |
| ----------- | ----------- |
| ![Screen Shot 2021-06-28 at 9 55 03 AM](https://user-images.githubusercontent.com/24237393/123659487-5bfb7700-d7f8-11eb-8972-05851e299fb6.png) | ![Screen Shot 2021-06-28 at 10 10 40 AM](https://user-images.githubusercontent.com/24237393/123660345-328f1b00-d7f9-11eb-9d54-56f62a17cc40.png) |



## Testing in `sage-lib`
1. Navigate to the [Page Headings component page](http://localhost:4000/pages/component/page_heading)
2. Verify that the help icon outputs a visually hidden span that is **not** empty and contains a description


## Testing in `kajabi-products`
(LOW) - This change affects all uses of the help icon in the Page Heading within `kajabi-products`. While this sounds like a breaking change, not a single instance of `label` was being used at the time of this PR. The following list are several areas that can be spot checked to verify that the help icon is outputting the description in the visually hidden span.
  - [ ] Products landing page
  - [ ] Offers landing page
  - [ ] Payment landing page
